src/status_im/ui/screens/wallet/send/db.cljs

```diff
@@ -18,10 +18,10 @@
 (spec/def ::camera-flashlight #{:on :off})
 (spec/def ::camera-permitted? boolean?)
 (spec/def ::in-progress? boolean?)
-(spec/def ::from-chat? boolean?)
+(spec/def ::from-chat? (spec/nilable boolean?))
 
 (spec/def :wallet/send-transaction (allowed-keys
                                      :opt-un [::amount ::to-address ::to-name ::amount-error ::password
-                                              ::waiting-signal? ::signing? ::transaction-id ::later?
+                                              ::waiting-signal? ::signing? ::id ::later?
```

Comment:
`transaction-id` is more clear

Reply:
there was a mix of id and transaction-id I just settled on id but yeah can switch to transaction-id

Reply:
the main reason I settled for id is because that is how the field is called in unsigned-transactions list